### PR TITLE
Walkthrough edits

### PR DESCRIPTION
**Pull request by josiblair**
## Description
Updates to "Initializing a K8s Cluster" and "Creating a k8s Cluster with Zarf" Walkthroughs
...

## Related Issue

Partial fix for #1377 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Other (security config, docs update, etc)

## Checklist before merging

- [X] Test, docs, adr added or updated as needed
- [X] [Contributor Guide Steps](https://github.com/defenseunicorns/zarf/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

File: docs/13-walkthroughs/4-creating-a-k8s-cluster-with-zarf.md [hunk 1]
```diff
@@ -1,18 +1,55 @@
...
+```bash
+zarf tools monitor
 
-## Walkthrough Prerequisites
-1. The [Zarf](https://github.com/defenseunicorns/zarf) repository cloned: ([`git clone` Instructions](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository))
-1. Zarf binary installed on your $PATH: ([Install Instructions](../3-getting-started.md#installing-zarf))
-1. An init-package built/downloaded: ([init-package Build Instructions](./0-using-zarf-package-create.md)) or ([Download Location](https://github.com/defenseunicorns/zarf/releases))
-1. `root` access on a Linux machine
+# Note you can press `0` if you want to see all namespaces and CTRL-C to exit
+```
```
Comment (Racer159) [inline docs/13-walkthroughs/4-creating-a-k8s-cluster-with-zarf.md:48]:
```suggestion
```

### Accessing the Cluster as a Normal User

By default the `k3s` component will only automatically provide cluster access to the `root` user to access the cluster as another user you can run the following to setup the `~/.kube/config` file:

```sh
# cp /root/.kube/config /home/otheruser/.kube
# chown otheruser /home/otheruser/.kube/config
# chgrp otheruser /home/otheruser/.kube/config
```
```

Comment (Racer159) [reply docs/13-walkthroughs/4-creating-a-k8s-cluster-with-zarf.md:48]: Well that borked the suggestion... But that is the rough gist...

**Review by Racer159 — CHANGES_REQUESTED** [review]
a few changes.

File: docs/13-walkthroughs/1-initializing-a-k8s-cluster.md [hunk 2]
```diff
@@ -1,54 +1,54 @@
...
+![Zarf Tools Monitor](../.images/walkthroughs/zarf_tools_monitor.png)
 
 ## Cleaning Up
 
-The [`zarf destroy`](../4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_destroy.md) command will remove all of the resources that were created by the initialization command. Since this walkthrough involved a kubernetes cluster that was already existing, this command will leave you with a clean cluster that you can either destroy or use for another walkthrough.
+The [`zarf destroy`](../4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_destroy.md) command will remove all of the resources that were created by the initialization command. This command will leave you with a clean cluster that you can either destroy or use for another walkthrough.
 
-```bash
+```sh
 zarf destroy --confirm
```
[the remark this answers is not in the document]
Comment (YrrepNoj) [reply docs/13-walkthroughs/1-initializing-a-k8s-cluster.md:53]: IMO, I prefer it the way it is, it makes it possible to copy the block and not have to mess around with deleting the first character in the line or having to be careful when highlighting the line to copy.

**Review by Racer159 — APPROVED** [review]
beautiful

**Pull request merged**